src/LazyOperations/Bloating.jl

```diff
@@ -26,8 +27,8 @@ struct Bloating{N, S<:LazySet{N}} <:LazySet{N}
     p::N
 
     function Bloating(X::S, ε::N, p::N=N(2)) where {N, S<:LazySet{N}}
-        @assert ε > zero(N) "bloating requires a distance > 0, but $ε was given"
         @assert p >= one(N) "bloating requires a norm >= 1, but $p was given"
+        @assert ε >= zero(N) || p > N(1) "negative bloating in 1-norm does not work"
```

Comment:
not sure about the impediment to construct Bloating with `p = 1` is (... "does not work" refers to some method(s) that the user may not or may not require)